repository-data/application/src/main/resources/hcm-config/namespaces/hee/contentCards.yaml

```diff
@@ -31,6 +32,7 @@ definitions:
             hipposysedit:path: hee:cards
             hipposysedit:primary: false
             hipposysedit:type: hippo:mirror
+            hipposysedit:validators: [required]
```

Comment:
Built a content/navigation cards validator (`uk.nhs.hee.web.validation.validator.ContentCardContentBlockValidator`) targetting only content blocks (so that it doesn't affect `hee:contentCards` based fields [e.g. `relatedContent` on `hee:guidance`, etc])